fitness/templates/fitness/single.html

```diff
@@ -2,56 +2,136 @@
-            <li>{{ resistance_set.reps }} Reps, {{ resistance_set.load }}lbs, {{ resistance_set.work }} work</li>
-          {% endfor %}
-        </ul>
-        {% if resistance_scheme.notes %}
-          <p>{{ resistance_scheme.notes }}</p>
-        {% endif %}
-      </li>
+    {% for resistance_set in resistance_scheme.resistance_set.all %}
+
+          <tr>
+            {% if forloop.first %}
+              <td>{{ resistance_scheme.exercise.title }} </td>
+            {% else %}
+              <td></td>
+            {% endif %}
+              <td>{{ resistance_set.reps }} @ {% if resistance_set.load == 0 %}BW{% endif %}{{ resistance_set.load }} </td>
+              <td>{{ resistance_set.work }} </td>
+              <td>{{ resistance_set.intensity }} </td>
+            </tr>
+
     {% endfor %}
-  </ul>
-
-
-  <h2>Cardio</h2>
-  <ul>
-    {% for cardio_scheme in workout.cardio_scheme.all %}
-      <li>
-        You did {{ cardio_scheme.exercise.title }}
-        <ul>
-          {# Interval #}
-          {% for interval in cardio_scheme.interval_set.all %}
-            <li>for {{ interval.action_time_seconds }} seconds with {{ interval.rest_time_seconds }} of seconds rest {{ interval.quantity }} times</li>
-          {% endfor %}
-
-          {# Distance #}
-          {% for distance in cardio_scheme.distance_set.all %}
-            <li>for {{ distance.distance }} {{ distance.measurement }} in {{ distance.time_seconds }} seconds</li>
-          {% endfor %}
-
-          {# Repetition #}
-          {% for repetition in cardio_scheme.repetition_set.all %}
-            <li>for {{ repetition.quantity }} times in {{ repetition.time_human }}</li>
-          {% endfor %}
-        </ul>
-      </li>
     {% endfor %}
-  </ul>
+          </table>
+    
+
+
+
+
+<!-- CARDIO -->
+
+
+<!-- NEED TO GET THIS ONLY APPEAR IF IT EXISTS -->
+{% if workout.cardio_scheme %} <h2>Cardio</h2> {% endif %}
+<!-- END HERE -->
+
+  {% for cardio_scheme in workout.cardio_scheme.all %}
+
+
+    {% for interval in cardio_scheme.interval_set.all %}
+
+    {# Interval #}
+
+      <caption>Interval</caption>
+      <table class="table table-striped">
+        <tr>
+          <th>EXERCISE</th>
+          <th>ACTION</th>
+          <th>RECOVERY</th>
+          <th>QUANTITY</th>
+        </tr>
+        <tr>
+          <td>{{ cardio_scheme.exercise.title }}</td>
+          <td>{{ interval.action_time_seconds }}</td>
+          <td>{{ interval.rest_time_seconds }}</td>
+          <td>{{ interval.quantity }}</td>
+        </tr>
+      {% endfor %}
+    </table>
+
+    <br>
+
+    {% for distance in cardio_scheme.distance_set.all %}
+
+    {# Distance #}
+  
+      <caption>Distance</caption>
+        <table class="table table-striped">
+          <tr>
+            <th>EXERCISE</th>
+            <th>DISTANCE</th>
+            <th>MEASUREMENT</th>
+            <th>TIME</th>
+          </tr>
+          <tr>
+            <td>{{ cardio_scheme.exercise.title }}</td>
```

Comment:
or just move the forloop to generate the detail rows and leave the rest inside an if statement